config/prov/prov_test.yaml

```diff
@@ -123,16 +123,21 @@ k8s_cortx_deploy:
     memory: 16
   clone_dir: "deploy-scripts"
   local_path_prov: "/mnt/fs-local-volume"
+  3rd_party_data_dir: "/var/data/3rd-party/"
+  3rd_party_dir: "/etc/3rd-party/"
   log_path: "/etc/cortx/log"
   custom_log_path: "/etc/cortx/custom/log"
   s3_max_start_timeout: 240
   setup_size: "small"
   protocol: "http://{}:{}"
+  https_protocol: "https://{}"
   https_port: 30443
   http_port: 30080
   control_port_https: 31169
   iface: "eth1"
   service_type: "LoadBalancer"
+  lb_count: 1
```

Comment:
so in case of loadbalancer this count will come from??

Reply:
Jenkins job